### ISSUE TITLE
private parking leads to misunderstatings

### ISSUE THREAD

**Issue by woodpeck** [gravitystorm/openstreetmap-carto#4984]
One of the most frequent "private property" complaints that reach the DWG is about parking: "People are parking on my private property and when confronted they claim the map indicates a car park!"

Frequently the car park will be correctly mapped as access=private or access=customers, but neither property owners nor drivers seem to mind.

I suggest that we:
* either drop the rendering of private car parking altogether (because those authorized to use it will know, and those not authorized don't have to know)
* or label it with "(private)" wherever it doesn't have a name
* or change the icon from a simple "P" to something with a little padlock 
* or anything else that makes it clearer than at present that you can't just park your car there.

### Expected behavior

The expected behaviour is that the map does not mislead users into believing there was a public car park when there isn't.

### Actual behavior

The actual behaviour is that many users - land owners as well as car operators - misleadingly believe our faint blue "P" was indicating public access. If this happens to a few dumb people then it is something we can ignore, but if it happens every day then there is a problem with the map.

### Screenshots with links illustrating the problem

(Sample screenshot - without a contrasting public "P" in the vicinity, there's nothing to suggest that this was not public parking.)

**Comment by imagico** [comment]
Yes, the current design of indicating access restricted POIs of various types with a lighter color symbol is a conveniently simple but not very intuitive approach to rendering.

> * either drop the rendering of private car parking altogether (because those authorized to use it will know, and those not authorized don't have to know)

I don't think this is a good idea in this categorical form.  There are huge private parking lots (like of factories for their workers) that would not be good to just hide from the map.  I would regard the 'P' not only as a _you can park here_ indicator but likewise as a _cars are parked here_ sign.  And the latter equally applies to public and private parkings.

> * or label it with "(private)" wherever it doesn't have a name

Does not work on a global multilingual map because (private) is English language.  Even if we'd spatially differentiate the style and use translations in all the languages of the world it would still be much better to have a language independent visualization.

> * or change the icon from a simple "P" to something with a little padlock

That is an idea worth exploring but we would need to combine it with a more differentiated visualization of the different access values (like we already discussed in #4717 and #4890).  Showing all access values we currently render with a lighter color ('private', 'no', 'customers', 'permit', 'delivery') with a padlock would be misleading.  But doing so for `access=private` (593k uses on `amenity=parking`) while leaving `access=customers` (482k uses) with the plain 'P' symbol would work - and provides a practically meaningful differentiation for the map user as well.

It would also be worth going through the other POI types we render with a lighter symbol with access restrictions if they could benefit from a similar differentiation.
